specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -4861,6 +4861,95 @@
         "authenticationType"
       ]
     },
+    "SapTableLinkedService": {
+      "x-ms-discriminator-value": "SapOpenHub",
```

Comment:
SapTable

Reply:
@Yiliu-microsoft  Please make sure to fix this before merge. It is interesting that tools didn't catch this. @shahabhijeet there was already another type with the same x-ms-discriminator-value, the verification builds on swagger don't catch it. Is it expected?